src/eventreceiver.ts

```diff
@@ -0,0 +1,47 @@
+import { Tracer } from './telemetry/traces';
+import { GlobalLogger as Logger } from './telemetry/logs';
+import { WorkflowFunction, WorkflowHandle, WorkflowParams } from './workflow';
+import { Transaction } from './transaction';
+import { MethodRegistrationBase } from './decorators';
+
+/*
+ * Info provided to an event receiver at initialization,
+ *  which contains the things that it needs to do its work
+ *  (retrieve decorated endpoints, and run new transactions / workflows)
+ */
+export interface DBOSExecutorEventReceiverInterface
```

Comment:
I think the name could be something like `DBOSExecutorContext` because it looks more like a context you could use from outside of the main DBOS server.

Reply:
That's a bit better.